src/js/tabs/ports.js

```diff
@@ -132,6 +132,8 @@ TABS.ports.initialize = function (callback, scrollPosition) {
 
         $(".tab-ports").addClass("supported");
 
+        const vcpPortIndentifier = 20;
```

Comment:
If this a constant, maybe is better uppercase, like `VCP_PORT_IDENTIFIER`. We have no rules defined for this but this way is clear later when used.

Reply:
@etracer65: Are you going to change this?

Reply:
Updated constant name to `VCP_PORT_IDENTIFIER`.